app/templates/views/guidance/using-notify/unsubscribe-links.html

```diff
@@ -12,18 +13,35 @@
   <h1 class="heading-large">Unsubscribe links</h1>
 
   <p class="govuk-body">You must let people opt out of receiving subscription emails.</p>
-  <p class="govuk-body">If you do not, recipients can report your emails as spam. If this happens we may stop you from using GOV.UK Notify.</p>
-  <p class="govuk-body">Transactional emails do not need to include an unsubscribe link.</p>
+  <p class="govuk-body">If recipients report your emails as spam we can stop you from using GOV.UK Notify.</p>
+ 
+  <p class="govuk-body">There are 2 ways to let recipients unsubscribe. We recommend you use both of them.</p>
+  <ol class="govuk-list govuk-list--number">
+    <li><a class="govuk-link govuk-link--no-visited-state" href=“#oneclick”>Add one-click unsubscribe to the email header</a>.</li>
+    <li><a class="govuk-link govuk-link--no-visited-state" href=“#emailbody”>Add a link to the body of your email</a>.</li>
+  </ol>
+
+ <p class="govuk-body">Transactional emails do not need to include an unsubscribe link.</p>
    <p class="govuk-body">The GOV.​UK Service Manual explains the difference between <a class="govuk-link govuk-link--no-visited-state"
     href="https://www.gov.uk/service-manual/design/sending-emails-and-text-messages#transactional-messages">transactional and subscription messages</a>.</p>
 
-  <p class="govuk-body">There are 2 ways to let recipients opt out:</p>
+  <h2 class="govuk-heading-m" id=“oneclick”>Add one-click unsubscribe to the email header</h2>
+  <h3 class="heading-small">If you use the Notify website to send emails</h3>
+  <p class="govuk-body">To add one-click unsubscribe:</p>
   <ol class="govuk-list govuk-list--number">
-    <li>Add an unsubscribe link to the body of your email.</li>
-    <li>Add one-click unsubscribe to the email header.</li>
+    <li>Go to the <b class="govuk-!-font-weight-bold">Templates</b>page.</li>
+    <li>Add a new email template or choose an existing email template.</li>
+    <li>Select <b class="govuk-!-font-weight-bold">Allow recipients to unsubscribe</b>.</li>
   </ol>
+  <p class="govuk-body">This will add a button in the email header. Recipients can opt out of unwanted emails by clicking on the button.
```

Comment:
```suggestion
  <p class="govuk-body">This will add a button in the email header. Recipients can ask to opt out of unwanted emails by clicking on the button.
```

I’m not 100% on the suggested wording – but I think we need to be clear that clicking the button won’t **actually** unsubscribe the recipient.

All it does is capture that they want to unsubscribe. The service still needs to download their report and take steps to ensure they stop sending emails to that user.

Reply:
Thanks Karl. I did wonder about this but tried to explain it in the paragraphs that follow on. 

I agree with tweaking this though so we're making every effort to not mislead users. 

We've used the wording 'unsubscribe requests' in the UI and further in the page. Replacing your suggested word 'ask' with 'request' would tie this all in nicely.